studips/src/component/PostModal.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import '../App.css';
+
+function PostModal({ isPostModalVisible }) {
+	return isPostModalVisible ? (
```

Comment:
use a more suited operator: if your "else" is null, prefer `&&` ;)

Reply:
Fixed :)